terraform.tfvars.example

```diff
@@ -4,7 +4,7 @@ cloudwatch_retention = 7
 
 # Cluster Autoscaler Helm Config
 cluster_autoscaler_helm_config = {
-  version = "9.28.0"
+  "version": "9.28.0"
```

Comment:
Why the version = "9.28.0" is changed with "version": "9.28.0"? What does this do for deployment, is this option compatible in both cases?

Reply:
Yes, that's the "unfortunate" side effect of qualitygate using hcl style formatting. Fortunately, it does not affect end-result in any way, I've tested it both from clean deployment and from already existing deployment (deleting helm deployment and using Helm to redeploy the chart). Code is public, so anyone can verify. I think this is the best compromise, we can keep qualitygate enabled and have default value that actually works.

Reply:
Yes, that's the "unfortunate" side effect of qualitygate using hcl style formatting. Fortunately, it does not affect end-result in any way, I've tested it both from clean deployment and from already existing deployment (deleting helm deployment and using Helm to redeploy the chart). Code is public, so anyone can verify. I think this is the best compromise, we can keep qualitygate enabled and have default value that actually works.

Reply:
If it's ok, then it might be best to use same style in variables.tf file default value so it is aligned with terraform.tfvars.example file?

Reply:
Done